src/api/mappers/visualNovelMapper.ts

```diff
@@ -6,15 +6,38 @@ import {
   VisualNovelTag,
   VisualNovel,
 } from '../../models/visualNovel';
+import { StaffRoles } from '../../utils/types/staffRoles';
+import { VisualNovelLength } from '../../utils/types/visualNovelLength';
 import {
   VisualNovelRelatedAnimeDto,
   VisualNovelRelatedDto,
   VisualNovelStaffDto,
   VisualNovelScreenshotDto,
   VisualNovelDto,
-} from '../dtos/ visualNovelDto';
+} from '../dtos/visualNovelDto';
 import { imageFlaggingFromDto } from './imageFlaggingMapper';
 
+/**
+ * Maps VN length to readable format.
+ * @param length Visual novel length.
+ */
+const mapVisualNovelLength = (length: number): VisualNovelLength => {
+  switch (length) {
+    case 1:
+      return VisualNovelLength.VeryShort;
+    case 2:
+      return VisualNovelLength.Short;
+    case 3:
+      return VisualNovelLength.Medium;
+    case 4:
+      return VisualNovelLength.Long;
+    case 5:
+      return VisualNovelLength.VeryLong;
+    default:
+      return VisualNovelLength.Medium;
+  }
+};
```

Comment:
I think `VisualNovelLength` enum should be used for switch cases, not for returning of values

Reply:
Check how you did it in `src/utils/types/roles.ts` 😄